com.fribbledom.telephant.json

```diff
@@ -0,0 +1,148 @@
+                },
+                {
+                    "type": "git",
+                    "url": "https://go.googlesource.com/xerrors",
+                    "commit": "9bdfabe68543c54f90421aeb9a60ef8061b5b544",
+                    "dest": "src/golang.org/x/xerrors"
+                },
+                {
+                    "type": "git",
+                    "url": "https://go.googlesource.com/sys",
+                    "commit": "5c8b2ff67527cb88b770f693cebf3799036d8bc0",
+                    "dest": "src/golang.org/x/sys"
+                },
+                {
+                    "type": "git",
+                    "url": "https://go.googlesource.com/mod",
+                    "commit": "e5e73c1b9c72835114eb6daab038373d39515006",
+                    "dest": "src/golang.org/x/mod"
+                },
+                {
+                    "type": "git",
+                    "url": "https://github.com/mattn/go-mastodon",
+                    "commit": "3e91c76504df99d2066afada755594ee8b554d1d",
+                    "dest": "src/github.com/mattn/go-mastodon"
+                },
+                {
+                    "type": "git",
+                    "url": "https://github.com/therecipe/qt",
+                    "commit": "5074eb6d8c41fdae5d1956af7c286160a40dd8fd",
+                    "dest": "src/github.com/therecipe/qt"
+                },
+                {
+                    "type": "git",
+                    "url": "https://github.com/gorilla/websocket",
+                    "commit": "c3e18be99d19e6b3e8f1559eea2c161a665c4b6b",
+                    "dest": "src/github.com/gorilla/websocket"
+                },
+                {
+                    "type": "git",
+                    "url": "https://github.com/sirupsen/logrus",
+                    "commit": "7ea96a3284ed9d25de5056804d60426cb75833c7",
+                    "dest": "src/github.com/sirupsen/logrus"
+                },
+                {
+                    "type": "git",
+                    "url": "https://github.com/BurntSushi/toml",
+                    "commit": "3012a1dbe2e4bd1391d42b32f0577cb7bbc7f005",
+                    "dest": "src/github.com/BurntSushi/toml"
+                },
+                {
+                    "type": "git",
+                    "url": "https://github.com/dustin/go-humanize",
+                    "commit": "afde56e7acacd811f6c94228c2c61af2b0e93158",
+                    "dest": "src/github.com/dustin/go-humanize"
+                },
+                {
+                    "type": "git",
+                    "url": "https://github.com/muesli/go-app-paths",
+                    "commit": "c184ea3ba2f517461749fca7a793d9609caff57d",
+                    "dest": "src/github.com/muesli/go-app-paths"
+                },
+                {
+                    "type": "git",
+                    "url": "https://github.com/muesli/telephant",
+                    "branch": "master",
```

Comment:
This needs to use specific commit/tag.

Reply:
Yeah, that's why there is a todo entry for it :)